packages/openneuro-server/datalad/dataset.js

```diff
@@ -290,7 +290,7 @@ export const addFileString = (datasetId, filename, mimetype, content) =>
 /**
  * Commit a draft
  */
-export const commitFiles = (datasetId, user) => {
+export const commitFiles = (datasetId, user, descriptionFieldUpdates) => {
```

Comment:
I think adding this here confuses the goal of commitFiles. Maybe a better way is to remove the createSnapshot handling here and accomplish the automatic snapshot in another way.